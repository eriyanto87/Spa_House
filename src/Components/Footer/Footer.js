import React from "react";

export default function Footer() {
  return (
    <div>
      <p>&copy; 2020</p>
    </div>
  );
}
