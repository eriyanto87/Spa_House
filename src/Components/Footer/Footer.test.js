import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import { BrowserRouter } from "react-router-dom";
import Footer from "./Footer";

describe("Footer component", () => {
  it("Footer Snapshot", () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <Footer />
        </BrowserRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(
      <BrowserRouter>
        <Footer />
      </BrowserRouter>,
      div
    );
    ReactDOM.unmountComponentAtNode(div);
  });
});
