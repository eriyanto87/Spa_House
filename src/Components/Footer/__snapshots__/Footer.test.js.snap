// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Footer Snapshot 1`] = `
<div>
  <p>
    Â© 2020
  </p>
</div>
`;
